build.sbt

```diff
@@ -166,6 +166,24 @@ lazy val circe = crossProject(JSPlatform, JVMPlatform)
 lazy val circeJVM = circe.jvm
 lazy val circeJS  = circe.js
 
+lazy val akkaJVM = project
+  .in(file("modules/akka"))
+  .dependsOn(coreJVM)
+  .settings(sharedSettings)
+  .settings(testSettings)
```

Comment:
The `testSettings` setup the munit test library. Not sure, if you want this when scalatest is indeed required. I'd really prefer to not have scalatest additionally in this project. But if this is dictated by akka (which I don't know), then there's no choice I guess. In this case, these `testSettings` are probably not needed or do even something we don't want?

Reply:
>> In this case, these testSettings are probably not needed 
Yes, you are right, I removed them.